.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  extends: ['react-app'],
+  ignorePatterns: ['dist', 'build', 'node_modules', '.storybook'],
```

Comment:
Please also check for this path: 'storybook-static'

Reply:
will be done in next PR